pkg/operator/ceph/csi/spec.go

```diff
@@ -756,6 +758,42 @@ func (r *ReconcileCSI) stopDrivers(ver *version.Info) error {
 	return nil
 }
 
+func (r *ReconcileCSI) deleteCSIOperatorResources(clusterNamespace string) {
+	csiCephConnection := &csiopv1a1.CephConnection{}
+
+	err := r.client.DeleteAllOf(r.opManagerContext, csiCephConnection, &client.DeleteAllOfOptions{ListOptions: client.ListOptions{Namespace: clusterNamespace}})
+	if err != nil && !kerrors.IsNotFound(err) {
+		logger.Errorf("failed to delete CSI-operator Ceph Connection %q. %v", csiCephConnection.Name, err)
+	}
+	logger.Infof("deleted CSI-operator Ceph Connection %q", csiCephConnection.Name)
```

Comment:
maybe put all the `logger.Info` in the else block. Since we are not returning the error, the code will show both `error` and `info` logs if there is any error.

Reply:
This is for all the delete operations below.